src/java.base/share/classes/java/util/BitSet.java

```diff
@@ -1037,7 +1037,7 @@ public int hashCode() {
         for (int i = wordsInUse; --i >= 0; )
             h ^= words[i] * (i + 1);
 
-        return (int)((h >> 32) ^ h);
+        return Long.hashCode(h);
```

Comment:
Here `>>` instead of `>>>` in original code seems to be a typo

Reply:
It is specified as `>>` in JavaDoc just above the implementation. As the algorithm is part of the public API and thus part of the specification, I don't think you can change it just here in the implementation; you'd need to at least submit a CSR for it.

Reply:
Good point! I'll then revert this change